packages/test/test-end-to-end-tests/src/test/rootDatastores.spec.ts

```diff
@@ -386,7 +427,11 @@ describeFullCompat("Named root data stores", (getTestObjectProvider) => {
 			},
 		);
 
-		it("getAliasedDataStoreEntryPoint only returns aliased data stores", async () => {
+		it("getAliasedDataStoreEntryPoint only returns aliased data stores", async function () {
+			// TODO: Re-enable after cross version compat bugs are fixed.
```

Comment:
#nit can we add the bug# here? For example, https://github.com/microsoft/FluidFramework/blob/d85c3ec19a5df110642761f6e61408d26ff9628c/experimental/dds/tree/src/test/utilities/PendingLocalStateTests.ts#L174 it's easier to track for the person who's supposed to fix it.